release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Native link element for app titles [ID 51524]
+
+The app title of the following apps: Admin, Catalog, Home, Marketplace & Sharing now use native link element, allowing users to open the apps in a new tab using middle-click or right-click options.
+
+### 9 December 2024 - Enhancement - Home - Updated terms of service and conditions for DaaS deployments [ID 41651]
+
+The link to the correct terms of service and conditions for DaaS deployments has been updated.
```

Comment:
Do you mean the link was incorrect previously? In that case, this should probably be indicated as a fix. Or if it wasn't incorrect, we should remove the word "correct", because that will otherwise give the wrong impression.

Reply:
Removed the word "correct" since the link was good but the (toc) page was updated.